diesel/src/expression/expression_methods/escape_expression_methods.rs

```diff
@@ -23,7 +23,7 @@ use types::VarChar;
 /// #     use diesel::insert;
 /// #     let connection = establish_connection();
 /// #     insert(&NewUser { name: "Ha%%0r".into() }).into(users)
-/// #         .execute(&connection).unwrap();
+/// #         .try!(execute(&connection));
```

Comment:
Sadly, macros (which `try!` is) can't be called like methods, so the entire expression needs to be wrapped. You can try...
```rust
try!(insert(..).into(..).execute(..));
```